### ISSUE TITLE
Cannot close the app via keyboard shortcut or the app menu

### ISSUE THREAD

**Issue by Jonuco** [marius-m/wt4#13]
When I try to quit the app via CMD+Q, nothing happens. Also, I cannot quit the app from the app menu - nothing happens:

<img width="261" alt="image" src="https://user-images.githubusercontent.com/9945334/99822177-8ea30a00-2b5b-11eb-8740-01669f2dedd7.png">

Version: 1.8.0
MacOS: 11.0.1

**Comment by marius-m** [comment]
Good catch. App has its internal shortcuts, which probably overrides system one's. I'll try to fix it on next upcoming release. Thanks! 👏

**Comment by marius-m** [comment]
Just had another idea what might be going on. As you quit the app, before it closes, it shows the dialog, to confirm the quit action, maybe something went haywire and did not show. Here how it looks like: 
<img width="811" alt="Screenshot 2020-12-05 at 08 27 37" src="https://user-images.githubusercontent.com/1816086/101235764-e9983d80-36d3-11eb-9125-858b859af597.png">

One way of closing it, press an 'ALT' button and then secondary button on an icon to 'Force quit'
![IMG_20201205_083012](https://user-images.githubusercontent.com/1816086/101235831-8955cb80-36d4-11eb-8631-e7073c34fd67.jpg)

I've done this, using kind of a trick of the framework. If it happens often, I'll have to reconsider of disabling this feature.

**Comment by marius-m** [comment]
Should not be an issue with [1.9.9](https://github.com/marius-m/wt4/releases/tag/1.9.9)
